code-challenges.test.js

```diff
@@ -10,40 +10,142 @@
+describe("mult3", () => {
+    it("takes in an array of numbers and returns an array with all the numbers multiplied by 3", () => {
+        const numbersArray1 = [6, 7, 8, 9, 10]
+        const numbers1Output = [18, 21, 24, 27, 30]
+        const numbersArray2 = [24, 27, 30, 33, 36]
+        const numbers2Output = [72, 81, 90, 99, 108]
+        expect(mult3(numbersArray1)).toEqual(numbers1Output)
+        expect(mult3(numbersArray2)).toEqual(numbers2Output)
+    })
+})
+    // ReferenceError: mult3 is not defined
+
 // b) Create the function that makes the test pass.
+    // Create the function called mult3 that takes in an array
+    // one parameter - array
+    // Iterate over array using .map() so that we can midify each value in the array
+        // multipply each value by three
+    // return mapped array
+const mult3 = (array) => {
+    return array.map(value => value * 3)
+}
+    //Test Suites: 1 passed, 1 total
+    //Tests:       1 passed, 1 total
+
 
-// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
+ // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+// const object1 = { number: 15 }
+// // Expected output: "15 is divisible by three"
+// const object2 = { number: 0 }
+// // Expected output: "0 is divisible by three"
+// const object3 = { number: -7 }
+// // Expected output: "-7 is not divisible by three"
+
+describe("divBy3", () => {
+    it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not", () => {
+        const object1 = { number: 15 }
+        const object1Output = "15 is divisible by three"
+        const object2 = { number: 0 }
+        const object2Output = "0 is divisible by three"
+        const object3 = { number: -7 }
+        const object3Output = "-7 is not divisible by three"
+        expect(divBy3(object1)).toEqual(object1Output)
+        expect(divBy3(object2)).toEqual(object2Output)
+        expect(divBy3(object3)).toEqual(object3Output)
+    })
+})
+    // ReferenceError: divBy3 is not defined
 
 // b) Create the function that makes the test pass.
+    // Create a function called divBy3
+    // one parameter = object
+    // Use dot notation to be able to pass through each object values in the for loop (object.number) because we were given objects and not an arrays
+        // I had a really hard time writing this because I used object instead of object.number and could not understand by I kept getting back undefined...but finally got it!!!
```

Comment:
Excellent work!!  It looks like you set up a beautiful for loop that is never actually used.  This problem works perfectly with just the conditional 😄